atomate/vasp/firetasks/glue_tasks.py

```diff
@@ -202,6 +203,52 @@ def run_task(self, fw_spec):
         return FWAction(stored_data=stored_data)
 
 
+@explicit_serialize
+class GetInterpolatedPOSCAR(FiretaskBase):
+    """
+    Grabs CONTCARS from two previous calculations to create interpolated
+    structure.
+
+    Required params:
+        start (str): name of fw for start of interpolation.
+        end (str): name of fw for end of interpolation.
+        this_image (int): which interpolation this is.
+        nimages (int) : number of interpolations.
+        autosort_tol (float): a distance tolerance in angstrom in which
+          to automatically sort end_structure to match to the closest
+          points in this particular structure.
+    """
+    required_params = ["start","end","this_image","nimages","autosort_tol"]
+
+    def run_task(self, fw_spec):
+
+        # make folder for poscar interpolation start and end structure files.
+        interpolate_folder = '/interpolate'
+        if not os.path.exists(os.getcwd()+interpolate_folder):
+            os.makedirs(os.getcwd()+interpolate_folder)
+
+            print (os.getcwd()+interpolate_folder)
+
+        # use method of GrabFilesFromCalcLoc to grab files from previous locations.
+        CopyFilesFromCalcLoc(calc_dir=None, calc_loc=self.get("start","default"), filenames=["CONTCAR"],
+                             name_prepend="interpolate/", name_append="_0").run_task(fw_spec=fw_spec)
+        CopyFilesFromCalcLoc(calc_dir=None, calc_loc=self.get("end","default"), filenames=["CONTCAR"],
+                             name_prepend="interpolate/", name_append="_1").run_task(fw_spec=fw_spec)
+
+        # assuming first calc_dir is polar structure for ferroelectric search
+
+        s1 = Structure.from_file("interpolate/CONTCAR_0")
```

Comment:
you defined a variable earlier called ``interpolate_folder``, use that instead of the hard-coded "interpolate"

Reply:
Done.